spec/configuration_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe ActionTracker::Configuration do
+  describe '#track_events flag' do
+    it 'default value is true' do
+      expect(ActionTracker::Configuration.new.track_events).to eq(true)
```

Comment:
No, I was sugesting something like this:

```ruby
  ActionTracker.configure do |config|
    config.track_events = false
  end

 expect(ActionTraker.configuration.track_events).to be_falsy
```

This way you would be testing that the `configure` class method from ActionTracker is working properly. What do you think?